frontends/ol-components/src/components/Checkbox/CheckboxChoiceField.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import { Checkbox, CheckboxProps } from "./Checkbox"
+import FormControl from "@mui/material/FormControl"
+import FormLabel from "@mui/material/FormLabel"
+import styled from "@emotion/styled"
+
+export type CheckboxChoiceFieldProps = {
+  label: React.ReactNode // We could make this optional, but we should demand one of (label, aria-label, aria-labelledby)
+  value?: string[]
+  name: string
+  choices: Omit<CheckboxProps, "name" | "onChange">[]
+  values?: string[]
+  onChange?: CheckboxProps["onChange"]
+  row?: boolean
+  className?: string
+}
+
+const Container = styled.div({
+  display: "flex",
+  gap: "32px",
+})
```

Comment:
Re the `row` prop: This isn't a big deal right now by any means, but the breakpoint hooks will cause us trouble if we ever render react server-side to send fuller HTML pages to the client. For this reason, I try to do the breakpoint stuff in css.

Here, we could do
```tsx
import type { Breakpoint } from "@mui/system"

const Container = styled.div<{columnBelow?: Breakpoint}>(({columnBelow})=>[
  {
    display: "flex",
    gap: "32px",
    flexDirection: "row"
  },
  columnBelow && {
    [theme.breakpoints.down(columnBelow)] {
      flexDirection: "column"
    }
  }
])
```

Reply:
The `columnBelow` thing here did not work, but just doing a standard `[theme.breakpoints.down("md")]` worked fine.